packages/react/src/themes/teams/components/Avatar/avatarStyles.ts

```diff
@@ -50,8 +50,9 @@ const avatarStyles: ComponentSlotStylesInput<AvatarProps, any> = {
   },
   status: ({ variables: v }): ICSSInJSStyle => ({
     position: 'absolute',
-    bottom: `-${v.statusBorderWidth}px`,
-    right: `-${v.statusBorderWidth}px`,
+    bottom: 0,
+    right: 0,
+    boxShadow: `0 0 0 ${pxToRem(v.statusBorderWidth)} ${v.statusBorderColor}`,
```

Comment:
Using a boxShadow to fake a border removes the need for fancy content + border width calculations.